tests/src/main/java/org/hibernate/beanvalidation/tck/tests/integration/cdi/defaultinjection/DefaultInjectionTest.java

```diff
@@ -62,8 +62,8 @@ public static WebArchive createTestArchive() {
 	@Test
 	@SpecAssertion(section = Sections.INTEGRATION_GENERAL_OBJECTSLIFECYCLE, id = "a")
 	@SpecAssertion(section = Sections.INTEGRATION_GENERAL_OBJECTSLIFECYCLE, id = "b")
-	@SpecAssertion(section = Sections.INTEGRATION_CDI_VALIDATORFACTORY, id = "a")
-	@SpecAssertion(section = Sections.INTEGRATION_CDI, id = "a")
+	@SpecAssertion(section = Sections.INTEGRATION_DEPENDENCYINJECTION_VALIDATORFACTORY, id = "a")
```

Comment:
I did not find the source of Sections, did you update it manually or is that automated based on spec anchors?

Reply:
It's a generated file.